opentracing-shim/src/test/java/io/opentelemetry/opentracingshim/testbed/clientserver/Client.java

```diff
@@ -38,7 +39,7 @@ public void send(boolean convertKeysToUpperCase) throws InterruptedException {
       if (convertKeysToUpperCase) {
         Message newMessage = new Message();
         for (Map.Entry<String, String> entry : message.entrySet()) {
-          newMessage.put(entry.getKey().toUpperCase(), entry.getValue());
+          newMessage.put(entry.getKey().toUpperCase(Locale.getDefault()), entry.getValue());
```

Comment:
not `Locale.ROOT` in this case?

Reply:
I'm not sure. My thought was to use `Locale.getDefault()` in places where the string is user defined, and not necessarily limited to basic latin characters. I might not be thinking about this right tho.

Reply:
hm. I see.  That's not a bad thought at all. of course, the default locale may or may not be relevant, but it might be relevant more often than Locale.ROOT?  🤔

Reply:
I think so.. after all, the `toUpperCase()` and `toLowerCase()` methods do default to `Locale.getDeafult()` instead of `Locale.ROOT`, suggesting that regionalization is (more?) often the right choice.